mesonbuild/interpreter/interpreter.py

```diff
@@ -2685,7 +2685,7 @@ def func_configure_file(self, node: mparser.BaseNode, args: T.List[TYPE_var],
             mlog.log('Configuring', mlog.bold(output), 'with command')
             res = self.run_command_impl(cmd,
                                         {'capture': True, 'check': True, 'env': EnvironmentVariables()},
-                                        True)
+                                        in_builddir=True)
```

Comment:
Is there a particular reason that passing this (non-ambiguous) argument as a kwarg is advantageous? I don't understand what difference this is supposed to make.

Reply:
Just a small pet peeve of mine, when passing just "true" you have to check signature to know what it means.

Reply:
Do you plan to change the entire codebase like this? My text editor shows function signatures anyway :D

Reply:
I can drop that commit if you want, not really important.